managed/ui/src/components/universes/UniverseForm/ClusterFields.js

```diff
@@ -724,6 +725,12 @@ export default class ClusterFields extends Component {
       updateFormField('primary.enableNodeToNodeEncrypt', event.target.checked);
       updateFormField('async.NodeToNodeEncrypt', event.target.checked);
       this.setState({ enableNodeToNodeEncrypt: event.target.checked });
+    } 
+    
+    // This condition will make sure if enableNodeToNodeEncrypt is false
+    // then enableClientToNodeEncrypt will set to false.
+    if(!event.target.checked) {
+      this.setState({ enableClientToNodeEncrypt: false});
```

Comment:
This `setState` call can be moved into the one above. Just check that the `event.target.checked` is false.
```
this.setState({
  enableNodeToNodeEncrypt: event.target.checked,
  enableClientToNodeEncrypt: this.state.enableClientToNodeEncrypt && event.target.checked
});    
```

Reply:
Done.